stats/stats_test.go

```diff
@@ -326,7 +376,11 @@ func (te *test) doUnaryCall(c *rpcConfig) (*testpb.SimpleRequest, *testpb.Simple
 
 	tCtx, cancel := context.WithTimeout(context.Background(), defaultTestTimeout)
 	defer cancel()
-	resp, err = tc.UnaryCall(metadata.NewOutgoingContext(tCtx, testMetadata), req, grpc.WaitForReady(!c.failfast))
```

Comment:
golang doesn't restrict columns for code so let's avoid the changes that are not relevant to the issue. Applicable everywhere.

Reply:
Remnant of an autoformatting mistake.